test/parallel/test-fs-symlink-dir.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+
+if (!common.canCreateSymLink())
+  common.skip('insufficient privileges');
+
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
```

Comment:
```suggestion
process.chdir(tmpdir)
```

Otherwise you mkdir and mklink at the node repo root.